FF/Pressure.C

```diff
@@ -16,6 +16,15 @@ void preciceAdapter::FF::Pressure::write(double* buffer, bool meshConnectivity,
 {
     int bufferIndex = 0;
 
+    if (this->locationType_ == LocationType::volumeCenters)
+    {
+        // set in Interface::addCouplingDataR/W
```

Comment:
What does this comment refer to?
Would a comment consistent with the "For every boundary patch of the interface" (i.e., "For every internal cell of the region") be more helpful to understand the intention?

Reply:
I added this comment for myself at an early stage and forgot to remove it later on. Fixed!